copier.yml

```diff
@@ -47,3 +60,8 @@ docs_option:
     help: Would you like docs?
 
 _subdirectory: "template"
+
+url:
+    type: str
+    default: "https://github.com/{{github_org}}/{{repo_name}}"
+    when: false
```

Comment:
I think we could also do with a `docs_url` that is `"https://{{github_org}}.github.io/{{repo_name}}"`